src/ProgressBar/__tests__/__snapshots__/ProgressBar.test.js.snap

```diff
@@ -172,7 +172,6 @@ exports[`<ProgressBar /> renders the progress bar properly 1`] = `
             iconRef={null}
             ml="smaller"
             name="calendar"
-            rotation={null}
```

Comment:
does this change any behavior?

Reply:
It doesn't appear to.  The prop was already optional, and `undefined` (or no prop) results in no rotation, so I considered `null` redundant.  But if there's any concern we can keep it for now.

Reply:
Okay, I put this default back too.  Now the remaining snapshot difference is just an implementation detail between `defaultProps` and inline defaulting.  The default value is `null`.